### PR TITLE
docs(trg): make dedicated container image notice mandatory

### PR DESCRIPTION
**Pull request by SebastianBezold**
## Description

This PR is a follow up to a [discussion on our mailing list](https://www.eclipse.org/lists/tractusx-dev/msg00306.html).
The discussion is about making a dedicated "Notice for Docker image" mandatory and no longer allow it to be a section in the top-level `README.md`.
So far, there has only been feedback pro change.

TL;DR from the discussion:

- Moving to a dedicated file is not a lot of work
- Can be used to better address target audience
- There is a character limit for DockerHub descriptions, which might cut off the mandatory notice section at the end of long repository README.md files

Fixes eclipse-tractusx/sig-infra#360

### CODE REVIEW

**Review by FaGru3n — APPROVED** [review]
LGTM only name for separate Docker `notice.md` could be discussed, i personally like the approach form "Country Risk" to name it "Docker_Notice.md"

File: docs/release/trg-4/trg-4-06.md [hunk 1]
```diff
@@ -77,17 +74,44 @@ As with all Docker images, these likely also contain other software which may be
 As for any pre-built image usage, it is the image user's responsibility to ensure that any use of this image complies with any relevant licenses for all software contained within.
 ```
 
-## Examples
+## Best practice and examples
 
-The following examples are shown as reference, to see already existing and complete versions of a 'Notice for docker images'.
-They **can not** be used for your product without modifications.
+- Use "Container images" as headline to link to the container image notice files
```
Comment (florianrusch-zf) [inline docs/release/trg-4/trg-4-06.md:79]:
```suggestion
- Use "Container images" as headline to link to the container image notice files from the repositories README.md file
```

File: docs/release/trg-4/trg-4-06.md [hunk 2]
```diff
@@ -77,17 +74,44 @@ As with all Docker images, these likely also contain other software which may be
...
+    readme-filepath: <path/to/your/notice.md>
+```
+
+### Examples
+
+The following examples are shown as reference, to see already existing and complete versions of a 'Notice for docker images'.
+They **can not** be used for your product without modifications.
 
-Good example for a dedicated notice file: [edc-controlplane-memory-hashicorp-vault](https://github.com/eclipse-tractusx/tractusx-edc/edit/main/edc-controlplane/edc-controlplane-memory-hashicorp-vault/notice.md)
+- [edc-controlplane-memory-hashicorp-vault](https://github.com/eclipse-tractusx/tractusx-edc/edit/main/edc-controlplane/edc-controlplane-memory-hashicorp-vault/notice.md)
```
Comment (florianrusch-zf) [inline docs/release/trg-4/trg-4-06.md:113]:
```suggestion
- [edc-controlplane-memory-hashicorp-vault](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-controlplane/edc-controlplane-memory-hashicorp-vault/notice.md)
```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

File: docs/release/trg-4/trg-4-06.md [hunk 3]
```diff
@@ -77,17 +74,44 @@ As with all Docker images, these likely also contain other software which may be
...
+  - Default port the service is listening on
+
+### GitHub workflow step
+
+Keeping the DockerHub description up-to-date with the latest notice content is crucial.
+The following workflow step can be used as part of the DockerHub publishing workflow to ensure that:
 
-Good example for notice integrated in toplevel `README.md`:
+```yaml
+# DockerHub publishing worklflow
```
[the remark this answers is not in the document]
Comment (SebastianBezold) [reply docs/release/trg-4/trg-4-06.md]: Thanks for spotting! All suggestions good and added via [8d76080](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/629/commits/8d76080b37b110c520e86977c7e48ee5e5f2a440)

**Review by FaGru3n — APPROVED** [review]
LGTM

**Review by carslen — APPROVED** [review]
LGTM

**Review by florianrusch-zf — APPROVED** [review]

**Pull request merged**